src/components/Card.jsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from 'gatsby'
+import React from 'react'
```

Comment:
This is admittedly a minor gripe, but let's import React first just to have our imports relatively consistent.